goodreads-scrape/script_add.py

```diff
@@ -0,0 +1,21 @@
+
+#!/usr/local/bin/python
+from olclient.openlibrary import OpenLibrary
+import olclient.common as common
+import requests
+import xml.etree.ElementTree as ET
+import sys
+
+isbn_13=str(sys.argv[1])
+API_KEY='1CSpGuaV4wRbkLdaVBYag&'
```

Comment:
I'd still have the user input their API key as a config variable instead of posting it here